eli5/sklearn_crfsuite/explain_weights.py

```diff
@@ -17,15 +17,21 @@ def explain_weights_sklearn_crfsuite(crf,
                                      top=20,
                                      target_names=None,
                                      targets=None,
-                                     feature_re=None):
-    """ Explain sklearn_crfsuite.CRF weights """
+                                     feature_re=None,
+                                     feature_filter=None):
+    """ Explain sklearn_crfsuite.CRF weights.
+
+    See :func:`eli5.explain_weights` for description of
+    ``top``, ``feature_names``,
```

Comment:
`target_names` and `targets` are also documented there, and there is no `feature_names` argument for explain_weights_sklearn_crfsuite